debian/control-armhf

```diff
@@ -0,0 +1,7 @@
+Package: bitwarden-rs
+Architecture: armhf
+Maintainer: Greizgh <greizgh+bitwardenrs@ephax.org>
+Priority: optional
+Version: 1.17.0-1
```

Comment:
This should not be hardcoded, see [here](https://github.com/greizgh/bitwarden_rs-debian/blob/master/build.sh#L54) how the version is handled.
We should have something similar for the architecture.

Reply:
Ah, good catch. I've removed the armhf specific control file and am letting the build script update it as with the version property instead.